contracts/src/validity/OPSuccinctL2OutputOracle.sol

```diff
@@ -309,7 +309,7 @@ contract OPSuccinctL2OutputOracle is Initializable, ISemver {
     /// @param _l2BlockNumber The L2 block number that resulted in _outputRoot.
     /// @param _l1BlockNumber The block number with the specified block hash.
     /// @param _proof The aggregation proof that proves the transition from the latest L2 output to the new L2 output.
-    /// @param _proverAddress The address of the prover that submitted the proof. Note: proverAddress is not required to be the msg.sender as there is no reason to front-run the prover.
+    /// @param _proverAddress The address of the prover that submitted the proof. Note: proverAddress is not required to be the tx.origin as there is no reason to front-run the prover.
```

Comment:
You should explain here under what conditions `tx.origin` is safe.

There are some attacks that can be made against `tx.origin`. As long as the relayers/proposers do not interact with untrusted contracts, `tx.origin` has equivalent security to `msg.sender`.